### PR TITLE
Update Gtimg.cn.xml

### PR DESCRIPTION
**Pull request by cschanaj**
The `exclusion` in this ruleset were added to pass Travis rules-test. IMHO, this is not ideal because it increases runtime overhead and memory usage with little values. I prefer to whitelist this ruleset from rules-test here.

### CODE REVIEW

**Comment by J0WI** [comment]
The tests are very valuable for maintaining rules. I would avoid disabling them. The exclusions are also there to prevent MCB issues.

File: src/chrome/content/rules/Gtimg.cn.xml [hunk 1]
```diff
@@ -79,7 +82,6 @@
 
 	<!--	MCB : https://github.com/EFForg/https-everywhere/pull/4591	-->
 	<target host="qzonestyle.gtimg.cn" />
-		<exclusion pattern="^http://qzonestyle\.gtimg\.cn/$" />
 	<rule from="^http://qzonestyle\.gtimg\.cn/(.+)\.(css|jpg|js|ico|png)"
```
Comment (J0WI) [inline src/chrome/content/rules/Gtimg.cn.xml]: There are also `(pdf|(doc|ppt|xls)x?)|swf)`

File: src/chrome/content/rules/Gtimg.cn.xml [hunk 2]
```diff
@@ -80,7 +90,7 @@
 	<!--	MCB : https://github.com/EFForg/https-everywhere/pull/4591	-->
 	<target host="qzonestyle.gtimg.cn" />
 		<exclusion pattern="^http://qzonestyle\.gtimg\.cn/$" />
-	<rule from="^http://qzonestyle\.gtimg\.cn/(.+)\.(css|jpg|js|ico|png)"
+	<rule from="^http://qzonestyle\.gtimg\.cn/(.+)\.(css|jpg|js|ico|png|pdf|(doc|ppt|xls)x?)|swf)"
```
Comment (J0WI) [inline src/chrome/content/rules/Gtimg.cn.xml]: The new extensions need test urls.

File: src/chrome/content/rules/Gtimg.cn.xml [hunk 3]
```diff
@@ -90,7 +90,7 @@
 	<!--	MCB : https://github.com/EFForg/https-everywhere/pull/4591	-->
 	<target host="qzonestyle.gtimg.cn" />
 		<exclusion pattern="^http://qzonestyle\.gtimg\.cn/$" />
-	<rule from="^http://qzonestyle\.gtimg\.cn/(.+)\.(css|jpg|js|ico|png|pdf|(doc|ppt|xls)x?)|swf)"
+	<rule from="^http://qzonestyle\.gtimg\.cn/(.+)\.(css|jpg|js|ico|png|pdf|(?:(?:doc|ppt|xls)x?)|swf)"
```
Comment (J0WI) [inline src/chrome/content/rules/Gtimg.cn.xml]: `(css|jpg|js|ico|png|pdf|(doc|ppt|xls)x?|swf)`

**Comment by cschanaj** [comment]
@J0WI travis is passing now. thanks!!

**Pull request merged**